semgrep-core/bin/Main.ml

```diff
@@ -878,7 +882,14 @@ let main () =
   in
 
   (* does side effect on many global flags *)
-  let args = Common.parse_options (options()) usage_msg (Array.of_list argv) in
+  let _args_tmp = Common.parse_options (options()) usage_msg (Array.of_list argv) in
```

Comment:
You can call it args. I ocaml you can do let x = a in let x = if t then x else b in

Reply:
Went with this suggestion!